src/nomadic/realtime/dashboard/builders.py

```diff
@@ -205,7 +215,10 @@ def _add_bedcov_row(self, bedcov_csv: str, regions: RegionBEDParser) -> None:
         # Dropdown of different statistics
         dropdown = dcc.Dropdown(
             id="bedcov-dropdown",
-            options=["mean_cov", "n_reads", "cov_gr100_per"],
+            options=[
+                {"label": t(option), "value": option, "title": t(f"{option}_tooltip")}
```

Comment:
I like the _tooltip nomenclature -- nice way to give flexibility while keeping everything linked to the original option.

Reply:
nice, thanks